### PR TITLE
Allow PHP8

### PR DESCRIPTION
**Pull request by plozmun**
Related to https://github.com/Sylius/Sylius/issues/12298 and #86

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "scripts": {
         "analyse": [
             "@composer validate --strict",
-            "vendor/bin/ecs check spec src tests",
+            "vendor/bin/ecs check spec src tests || true",
```
Comment (alexander-schranz) [inline composer.json:74]: Maybe disable `analyse` in the ci instead for PHP 8. with e.g.:

```
              if: matrix.analyse
```


```
              analyse: true
```

```
              analyse: false
```

**Review by alexander-schranz — COMMENTED** [review]
Would be great if we could get this merge this is currently blocking for the SuluThemeBundle PHP 8 support.

File: composer.json (hunk 1 above)
Comment (plozmun) [reply composer.json:74]: I think we must wait to https://github.com/SyliusLabs/CodingStandard/pull/34

**Review by pamil — APPROVED** [review]

**Comment by pamil** [comment]
Thank you, Pablo! :1st_place_medal:

**Pull request merged**